### ISSUE TITLE
[Bug Report][3.1.2] v-pagination aria label has missing/incorrect page number

### ISSUE THREAD

**Issue by Chris-McQuiggan** [vuetifyjs/vuetify#16555]
### Environment
**Vuetify Version:** 3.1.2
**Vue Version:** 3.2.45
**Browsers:** Chrome 109.0.0.0
**OS:** Windows 10, Mac OSX

### Steps to reproduce
For incorrect numbering:
1. Open reproduction link
2. Open dev tools
3. Reduce window width until the pagination reduces the number of rendered buttons (shows ellipsis)
4. Inspect the last number in the DOM (6)
5. OBSERVE aria-label="Goto Page 5"

For missing page numbers:
1. Create a new project locally using yarn create vuetify: typescript, vue router, pinia, eslint, yarn and scss
2. Start a dev server
3. Render the v-pagination component
4. Inspect the page buttons in the DOM
5. OBSERVE aria label is "Goto Page "

### Expected Behavior
aria label to be "Goto Page 6"

### Actual Behavior
aria label is "Goto Page 5"

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNp9kMtOwzAQRX9l8CYgNbZ4LqIWlR1/wKLuIiTT1DR+yJ4UoSr/jp0EKbSCne/c8Zk7szmxF+f4sUNWsCWhdm1J+CwNwPKYl84Nz0HoUplJDdqVjTIlKWugaNE0tF9J9iSZ+PkiZn+SGGlLMZsSZai8cgQBqUu20s56ghN43EEPO281ZDFelporawKBDg2skn+dvWLbWnizvq2vspsEH3ERxBZsZMUQjn8Ea+KGp5RFTkaQrIChkmpxRtKS7YlcKITojDs0vLJarKMnfGdIacxrq9f3/I4/PIpaBZrXOQadv3v7GdDHiZItZnBSu6+/Bkx25N7yuxE7lRLyAiUi/4g+92hq9Oj/zX3W+yv7mXeRP83spenjLVtlDuHsjFUYTrjZppZ+wUZ0bB72YNtvQybJAg==)

### Other comments
The incorrect page numbers I can seen consistently across computers whereas the missing page number has only been observed on MacOS.
